src/header-navigation/examples.js

```diff
@@ -0,0 +1,265 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+/* eslint-disable react/display-name*/
+
+import * as React from 'react';
+import {Button} from '../button';
+import {StatefulMenu as Menu} from '../menu';
+import {StatefulSelect as Search} from '../select';
+import {styled} from '../styles';
+import COLORS from '../select/examples-colors';
+
+import {HeaderNavigation, ALIGN, NavigationItem, NavigationList} from './index';
+import tests from './examples-list';
+import {TYPE} from '../select';
+
+const Link = styled('a', props => ({
+  ':hover': {
+    color: props.$theme.colors.primary400,
+  },
+}));
+
+let ITEMS = [];
+for (let i = 0; i < 100; i++) {
+  ITEMS.push({label: `Item ${i}`});
+}
+class MenuContainer extends React.Component<
+  {isOpen: boolean, align: string},
+  {isOpen: boolean},
+> {
+  static defaultProps = {
+    isOpen: false,
+    align: '',
+  };
+  state = {
+    isOpen: this.props.isOpen,
+  };
+  render() {
+    return (
+      <React.Fragment>
+        <span
```

Comment:
For any icon that triggers a menu we should probably have it within the tab order `tabIndex=0` and provide it a human readable label `aria-label="Navigation menu"`

Reply:
There are also some accessibility issues with this. I see we have the `jsx-a11y` plugin added as a dependency, but it isn't enabled in `eslint`.

@gergelyke is there any specific reason this isn't enabled right now?

Reply:
not that I am aware of - would you mind sending a PR @austin94 that enables it?

Reply:
@austin94 added code